google/cloud/security/common/data_access/sql_queries/create_tables.py

```diff
@@ -73,3 +73,12 @@
         PRIMARY KEY (`id`)
     ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 """
+
+CREATE_GSUITE_ACCOUNT_GROUPS_TABLE = """
+    CREATE TABLE `{0}` (
+        `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
```

Comment:
Are there other attributes we like to save?  i.e. "name"

BTW, what is the resource description for the group response?  is there a link?